Modix.Data/ModixContext.cs

```diff
@@ -127,6 +129,13 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 .Entity<DesignatedChannelMappingEntity>()
                 .Property(x => x.ChannelDesignation)
                 .HasConversion(new EnumToStringConverter<ChannelDesignation>());
+
+            modelBuilder.Entity<MessageEntity>()
+                .HasIndex(x => x.MessageId)
+                .IsUnique();
+
+            modelBuilder.Entity<MessageEntity>()
+                .HasIndex(x => new { x.GuildId, x.UserId });
```

Comment:
Remove this after adding an `Author` navigation property, as it'll be redundant.